### PR TITLE
remove the Unix executable attribute from non-executable files

### PR DESCRIPTION
**Pull request by mirabilos**
it most likely got added by people using git on MS-DOS/Windows

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Travie CI failed for unrelated reasons

**Comment by Jojo-Schmitz** [comment]
It is a shell script regardless, and keeping those 2 execute attributes doesn't harm and lets them getting executed as shell scripts. If if ain't broke, don't fix

**Comment by mirabilos** [comment]
Joachim Schmitz dixit:

>It is a shell script regardless, and keeping those 2 execute
>attributes doesn't harm and lets them getting executed as shell
>scripts.

No, they don’t really.

I’ll push a modified commit in which I’ll add a proper shebang
to build/travis/job_macos/generateGitLog.sh as to let it keep
its +x attribute.

File: build/travis/job_macos/generateGitLog.sh [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env bash
```
Comment (Jojo-Schmitz) [inline build/travis/job_macos/generateGitLog.sh]: Ah, right, a simple `#!/bin/sh` won't do on any platform where /bin/sh is not bash.

**Comment by shoogle** [comment]
> [the executable bit] most likely got added by people using git on MS-DOS/Windows

or by someone on a Unix system (including Linux and macOS) if their Git repository is stored on a filesystems that don't support Unix permissions, such as FAT or NTFS, as these are mounted `+rwx` by default.

Regardless, when using Git on a non-Unix file system (either within Windows or another OS), see these StackOverflow answers to solve this problem for [__new__](https://stackoverflow.com/a/1580644) and [__existing__](https://stackoverflow.com/a/40979016) files.

**Pull request merged**